src/i18n/messages/en.messages.json

```diff
@@ -12,42 +12,33 @@
       "removeLabel": "Remove {suggestion} from history queries",
       "title": "History Queries"
     },
-    "querySuggestions": {
-      "title": "Suggestions"
-    },
     "nextQueries": {
       "title": "Next Queries"
     },
     "nextQueriesGroup": {
       "title": "You may be interested",
       "message": "This is what other shoppers searched after <span class=\"x-text x-text--bold\">\"{query}\"</span>"
     },
-    "identifierResults": {
-      "title": "SKU Search"
-    },
     "recommendations": {
       "title": "Top Clicked"
     },
     "facets": {
       "gender": "GENDER",
       "categoryPaths": "CATEGORIES",
-      "price": "PRICE",
-      "usage": "USAGE",
-      "season": "SEASON"
+      "price": "PRICE"
     },
     "filters": {
-      "all": "All"
+      "all": "Show more"
+    },
+    "partialResults": {
+      "message": "VIEW ALL"
```

Comment:
This is not the message for the partials in the designs. If we should change it in this task it should be:

```suggestion
      "message": "View all results ({totalResults})"
```

![image](https://user-images.githubusercontent.com/7124620/175527533-debb1ab5-51ff-4a74-9446-81029b691c0b.png)